src/screen/TouchPadScreen.js

```diff
@@ -83,46 +89,59 @@ const TouchPadScreen = ({ navigation: { navigate }, route }) => {
     }
   });
 
-  drawingGesture.onUpdate((event) => {
-    xPosition.current = parseInt(event.absoluteX);
-    yPosition.current = parseInt(event.absoluteY);
+  drawingGesture.onStart((event) => {
+    startXPosition.current = event.absoluteX;
+    startYPosition.current = event.absoluteY;
+    prevXPosition.current = startXPosition.current;
+    prevYPosition.current = startYPosition.current;
+  });
 
-    const xtest = event.absoluteX - xPosition.current;
-    const ytest = event.absoluteY - yPosition.current;
+  drawingGesture.onUpdate((event) => {
+    const changeX = prevXPosition.current - event.absoluteX;
+    const changeY = prevYPosition.current - event.absoluteY;
+
+    if (Math.abs(changeX) > 50 || Math.abs(changeY) > 50) {
+      prevXPosition.current = event.absoluteX;
+      prevYPosition.current = event.absoluteY;
+      const radian = Math.atan2(changeY, changeX);
+      const degree = (radian * 180) / Math.PI;
```

Comment:
```suggestion
      prevXPosition.current = event.absoluteX;
      prevYPosition.current = event.absoluteY;
      
      const radian = Math.atan2(changeY, changeX); 
      const degree = (radian * 180) / Math.PI;
```

const라는 변수의 선언문과 위의 변수 재할당부분은 개행을 나누어주는것이 좋아보입니다.